bin/reth/src/commands/stage/dump/execution.rs

```diff
@@ -10,7 +10,8 @@ use reth_node_ethereum::EthEvmConfig;
 use reth_primitives::stage::StageCheckpoint;
 use reth_provider::{ChainSpecProvider, ProviderFactory};
 use reth_revm::EvmProcessorFactory;
-use reth_stages::{stages::ExecutionStage, Stage, UnwindInput};
+use reth_stages::stages::ExecutionStage;
+use reth_stages_api::{Stage, UnwindInput};
```

Comment:
I'd like to reexport api types from stages, this way we don't need to change anything here

Reply:
Done.